src/engine/RuntimeInformation.cpp

```diff
@@ -105,7 +106,11 @@ double RuntimeInformation::getOperationTime() const {
   } else {
     auto result = totalTime_;
     for (const RuntimeInformation& child : children_) {
-      result -= child.totalTime_;
+      // The `totalTime_` does not account for the time spent during
+      // query planning.
```

Comment:
This comment is hard to understand for multiple reasons. Let me elaborte a bit because it touches upon a meta topic:

1. The relation to the code is not clear
2. The expression "does not account for" is not clear
3. The reference to `totalTime_` is ambiguous (of the operation or of the child)

I think it's better to briefly(!) describe what a code block does, mentioning special stuff. In this case, I would move the comment a the beginning of the else and say what it does, namely computing the time for only this operation, by taking the total time of the subtree and subtrating the total times of the children, unless they were completed during query planning.

You will probably argue that this is just paraphrasing the code. But that is only somewhat true. For example, one has to look up what exactly `totalTime_` measures. The suggested comment contains a little redundancy in important places.

Also, the cost of writing comments with a little redundancy is practically zero.

Reply:
1. I agree with  the concrete suggestions concerning the formulation of that comment.
2. 

> I think it's better to briefly(!) describe what a code block does,
I think we agree to disagree.

3. The cost of writing redundant comments (this includes redundancy between comments in different places as well as redundancy between comments and code (where the code is not implemented in a "strange" way).
The cost of maintaining such redundant comments over time is enormous to almost impossible if it gets out of hand. Comments that paraphrase the code have the additional hazard that people tend to read the comment and then think they understand the code. But I also know that we do not agree on this last argument.

5. The what that function does is stated in the function's documentation in the header. Did you check that?